### PR TITLE
bbox docs

### PR DESCRIPTION
**Pull request by Robinlovelace**
Still figuring-out best way of scaling it without moving the centre.

### CODE REVIEW

File: R/bbox.R [hunk 1]
```diff
@@ -62,7 +62,12 @@ bbox.MtrxSetSetSet = function(obj) {
 #' @examples
 #' a = st_sf(a = 1:2, geom = st_sfc(st_point(0:1), st_point(1:2)), crs = 4326)
 #' st_bbox(a)
-#' st_as_sfc(st_bbox(a))
+#' b = st_as_sfc(st_bbox(a))
+#' b_larger = b * 1.1
```
Comment (edzer) [inline R/bbox.R]: this functionality has nothing to do with st_bbox: why introduce it here?

Comment (Robinlovelace) [reply R/bbox.R]: To show people how you can work with bbs to meet a common need: to increase or decrease the 'field of view'. In short: user-friendliness.

Comment (Robinlovelace) [reply R/bbox.R]: But agree this example may not best demonstrate that (hence it being work in progress - just an idea).

**Comment by Robinlovelace** [comment]
OK I've updated it to actually demonstrate scaling and shifting the bb which is surely useful for at least some users... Any constructive review comments, or event just 'not interested ditch it' welcome.

**Comment by ratnanil** [comment]
Has this PR been ditched? Working with bounding boxes should def. be introduced somewhere. It's a very common task and somewhat obscurely documented (I keep looking up https://github.com/r-spatial/sf/issues/572#issuecomment-347602717). Or am I missing something?

**Pull request closed without merging**